app/templates/components/reminders-list.hbs

```diff
@@ -1,5 +1,11 @@
 <h1>{{title}}</h1>
```

Comment:
That's coming from reminders.hbs. Definitely not necessary but just wanted to try to pass it through...